examples/ELZ-ReferralServiceRequest-01.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ServiceRequest xmlns="http://hl7.org/fhir">
+    <id value="ELZ-ReferralServiceRequest-01"/>
+    <meta>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ELZ-ReferralServiceRequest"/>
+        <tag>
+            <system value="http://terminology.hl7.org/CodeSystem/common-tags"/>
+            <code value="actionable"/>
+        </tag>
+    </meta>
+    <status value="active"/>
+    <intent value="order"/>
+    <category>
+        <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.4.30.69"/>
+            <code value="1"/>
+            <display value="Verwijsbrief"/>
+        </coding>
+    </category>
+    <subject>
+        <reference value="Patient/ELZ-ReferralServiceRequest-01-Patient-01"/>
+        <type value="Patient"/>
+        <display value="Marijne van XXX_Prins"/>
+    </subject>
+    <authoredOn value="2022-08-12"/>
```

Comment:
The corresponding Composition has a `.date` that is 2 months later than the `.authoredOn` of this ServiceRequest. Does that make sense functionally?

Reply:
Yes, I think it's not logical if the dates are not the same. The Envelop and Kern are created at the same time. So I've changed this to the current (release) date.